mlir/include/mlir/Dialect/MIOpen/XdlopsCodeSelection.h

```diff
@@ -663,6 +677,34 @@ struct XdlopsCodeSelection {
       k = 2;
       cycles = 8;
       k_base = 2;
+    } else if (mfmaInstr == "mfma_i32_32x32x8i8") {
+      group_size = 4;
+      num_groups_blk = 4;
+      num_regs_blk = group_size * num_groups_blk;
+      num_threads_blk = 32;
+      wave_size = 64;
+      num_input_blks = wave_size / num_threads_blk;
+      num_output_blks = 1;
+      num_regs_xdlops = num_regs_blk * num_output_blks;
+      m = 32;
+      n = 32;
+      k = 8;
+      cycles = 64;
+      k_base = 4;
+    } else if (mfmaInstr == "mfma_i32_16x16x16i8") {
+      group_size = 4;
+      num_groups_blk = 1;
+      num_regs_blk = group_size * num_groups_blk;
+      num_threads_blk = 16;
+      wave_size = 64;
+      num_input_blks = wave_size / num_threads_blk;
+      num_output_blks = 1;
+      num_regs_xdlops = num_regs_blk * num_output_blks;
+      m = 16;
+      n = 16;
+      k = 16;
+      cycles = 32;
+      k_base = 4;
```

Comment:
@zjing14 I want to make sure you agree with those.

Reply:
LGTM